custom_components/tuya_local/devices/whm04_doorbell.yaml

```diff
@@ -117,31 +120,21 @@ secondary_entities:
       - id: 6
         type: boolean
         name: switch
-  - entity: sensor
-    name: Button
-    category: diagnostic
-    dps:
-      - id: 10
-        type: integer
-        name: sensor
-        optional: true
-        persist: false
-      - id: 5
-        type: base64
-        name: name
-        optional: true
-        persist: false
   - entity: binary_sensor
     name: Ringing
     class: sound
-    category: diagnostic
     dps:
-      - id: 10
-        type: integer
+      - id: 5
+        type: base64
         name: sensor
         optional: true
         persist: false
         mapping:
           - dps_val: null
             value: false
           - value: true
+      - id: 5
```

Comment:
This will not work - the same dp cannot be repeated within the same entity.  That is why the sensor and binary_sensor were originally separated.